build/pdf.worker.js

```diff
@@ -64698,7 +64698,7 @@ class Execute extends _xfa_object.XFAObject {
   constructor(attributes) {
     super(TEMPLATE_NS_ID, "execute");
     this.connection = attributes.connection || "";
-    this.executeType = (0, _utils.getStringOption)(attributes.executeType, ["import", "remerge"]);
+    this.executeType = (0, _utils.getStringOption)(attributes.executeType, ["imp" + "ort", "remerge"]);
```

Comment:
why was this changed? isn't our regex looking for `import(` or `import `?

Reply:
`\b(import)\b` was matching this one!